src/plugins/explore/public/application/pages/traces/trace_details/public/gantt_chart_vega/gantt_chart_vega.scss

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+.gantt-chart {
```

Comment:
i fear this is too generic of a name. we usually namespace our classnames to avoid collisions. can we do something like `exploreGanttChart` or something? Thats been our convention (to add `explore` prefix to all of our classNames)

Reply:
Updated all namespace to use the correct naming convention
```.exploreGanttChart ```